pkg/apis/pipeline/v1beta1/provenance.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+// Provenance contains all the information that needs to be recorded in a provenance i.e. ConfigSource
+type Provenance struct {
+	// Source identifies where the task/pipeline file came from.
+	Source *ConfigSource `json:"source,omitempty"`
+}
+
+// ConfigSource records where the task/pipeline file came from.
+type ConfigSource struct {
+	// URI indicates the identity of the source of the config.
+	// https://github.com/in-toto/attestation/blob/main/spec/field_types.md#ResourceURI
```

Comment:
```suggestion
	// Field Definition: https://github.com/in-toto/attestation/blob/main/spec/field_types.md#ResourceURI
```

Reply:
I thought this is an example at first glance

Reply:
Sounds good! Changed. Thanks @Yongxuanzhang ~